SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestReplicationConditions/TestReplicationConditionsActor.cpp

```diff
@@ -222,7 +231,40 @@ void ATestReplicationConditionsActor_Physics::SpawnDynamicComponents()
 		SpawnDynamicComponent<UTestReplicationConditionsComponent_Physics>(TEXT("DynamicTestReplicationConditionsComponent_Physics"));
 }
 
+void ATestReplicationConditionsActor_Physics::InitFakePhysics()
+{
+	// This setup is required to have Unreal attempt to replicate any physics information, including Actor.ReplicatedMovement.
+	// Actor.ReplicatedMovement contains bRepPhysics, which is necessary to replicate to clients so that
+	// FSpatialConditionMapFilter can check it when determining the value of the COND_*Physics conditions.
+	if (UPrimitiveComponent* RootPrimComp = Cast<UPrimitiveComponent>(GetRootComponent()))
+	{
+		if (FBodyInstance* BodyInstance = RootPrimComp->GetBodyInstance())
+		{
+			if (BodyInstance->BodySetup == nullptr)
+			{
+				BodySetup = NewObject<UBodySetup>(this, NAME_None);
+				BodySetup->CollisionTraceFlag = CTF_UseSimpleAndComplex;
+				BodySetup->BodySetupGuid = FGuid::NewGuid();
+				BodyInstance->BodySetup = BodySetup;
+
+#if WITH_PHYSX
```

Comment:
what happens if WITH_PHYSX is disabled?

Reply:
I just put this in here because I saw this types wrapped in the same ifdefs elsewhere in the code, but if I go with your suggestion of making a fake component that makes it easier to trick Unreal into replicating physics stuff, I won't need any of this.